src/libcore/num/wrapping.rs

```diff
@@ -12,13 +12,13 @@
 #![unstable(feature = "wrapping", reason = "may be removed or relocated",
             issue = "27755")]
 
-pub use intrinsics::{add_with_overflow, sub_with_overflow, mul_with_overflow};
+use intrinsics::{ add_with_overflow, sub_with_overflow, mul_with_overflow };
```

Comment:
@bluss because it seemed like someone forgot and left it there. There's no reason for it to be there. It just kinda stank of code smell. I can put it back if you'd like?